Aspen-Discovery/aspen-discovery
# feat: usage data download

This patch allows admins to download the content of the usage graphs found within the admin dashboard. If it is approved, the feature can then be added to the usage graphs of Summon and ILS, as well as those of any other dashboards to which usage graphs will be added in the future in further PRs.

Clicking the 'Export as CSV' button triggers a fetch of the most recent data for the usage graph from the database as well as its download as a CSV file. A warning message to the user indicates that the data in the file may be more recent than the on-screen data (and result in discrepancies), unless they choose to reload the page immediately after the download.

Test plan:

- once logged in as an Aspen admin, navigate to Aspen Administration > System Reports > Usage Dashboards
- navigate to the various usage graphs within, including section-wide graphs such as 'General Usage'
- notice the added 'Export as CSV' button, and the accompanying warning message
click 'Export as CSV'
- immediately refresh the page
- check that the file has been downloaded
- check that the values within the CSV match the ones in 'raw data table'
